crates/resvg/tests/integration/main.rs

```diff
@@ -55,10 +57,14 @@ pub fn render(name: &str) -> usize {
     );
     resvg::render(&tree, render_ts, &mut pixmap.as_mut());
 
-    // pixmap.save_png(&format!("tests/{}.png", name)).unwrap();
-    // Command::new("oxipng")
-    //     .args(["-o".to_owned(), "6".to_owned(), "-Z".to_owned(), format!("tests/{}.png", name)])
-    //     .output().unwrap();
+    if !Path::new(&png_path).exists() {
```

Comment:
Let's revert to the old implementation. I want this behavior to be manually trigger-able.

Reply:
What do you think about what I did now? Then you can simply do `REPLACE="" cargo test` to indiciate that you want to replace reference images. This way, you don't always have to comment/uncomment it.

Or should I still revert completely?

Reply:
It's fine.